Sources/protoc-gen-swift/MessageFieldGenerator.swift

```diff
@@ -116,7 +116,7 @@ class MessageFieldGenerator: FieldGeneratorBase, FieldGenerator {
                 p.print("\(comments)\(visibility)var \(swiftName): \(swiftStorageType) = \(swiftDefaultValue)")
             }
         }
-
+        return
```

Comment:
Can we delete the rest of the code here and remove the return?

Reply:
yes, updated it